<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMS_App</class>
 <widget class="QMainWindow" name="WMS_App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuItems">
    <property name="title">
     <string>Items</string>
    </property>
    <addaction name="actionItem_Search"/>
   </widget>
   <widget class="QMenu" name="menuConfigurations">
    <property name="title">
     <string>Configurations</string>
    </property>
    <addaction name="actionUsers"/>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuItems"/>
   <addaction name="menuConfigurations"/>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionItem_Search">
   <property name="text">
    <string>Item Search</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
